app/src/main/res/layout/dashboard_page.xml

```diff
@@ -120,80 +120,201 @@
                 android:layout_marginRight="8dp"
                 android:layout_marginBottom="8dp">
 
+                <!-- Existing LinearLayout inside the CardView -->
                 <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
                     android:orientation="vertical"
                     android:padding="8dp">
 
+                    <!-- Your existing TextViews in the first LinearLayout -->
                     <TextView
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:text="Sales and Sold Items"
                         android:textSize="11sp"
-                        android:textStyle="bold" />
+                        android:textStyle="bold"
+                        android:textColor="@color/black"/>
 
                     <TextView
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:text="*Current vs Previous"
                         android:textSize="7sp" />
```

Comment:
Remove this TextView "*Current vs Previous", it's causing confusion to the reader.